src/libsync/configfile.cpp

```diff
@@ -1118,7 +1118,7 @@ void ConfigFile::setAutomaticLogDir(bool enabled)
 
 QString ConfigFile::logDir() const
 {
-    const auto defaultLogDir = QString(configPath() + QStringLiteral("/logs"));
+    const auto defaultLogDir = QString(QStandardPaths::writableLocation(QStandardPaths::StateLocation) + QStringLiteral("/logs"));
```

Comment:
before we change the log location, existing log files should be migrated to the new place
otherwise people may loose logs that may be important for them

Reply:
Is `Application::setupLogging()` is the good place for log migration routine? Looks like it's entry point for `logDir()` usage.